makefile.shade

```diff
@@ -321,11 +276,11 @@ var buildTarget = "compile"
                 {
                     GitCommand(repo, string.Format("add -f {0}", gitFilePath));
                 }
-            } 
+            }
 
-            GitCommand(repo, "commit -am \"Updating json files to pin versions and build.cmd to pin KoreBuild and DNX\""); 
+            GitCommand(repo, "commit -am \"Updating json files to pin versions and build.cmd to pin KoreBuild and DNX\"");
 
-            GitCommand(repo, string.Format("push origin {0}:{0} -f", branch));
+            GitCommand(repo, "push origin master:master -f");
```

Comment:
Yeah - I'm not super certain what the utility of the flag is. Seems like one more place for things to go bad (since we force push to the branch) plus we don't actually pin anything apart from the master branch (as yet).